Dockerfile

```diff
@@ -3,15 +3,21 @@
 # VERSION	0.1
 
 FROM debian:jessie-slim
-MAINTAINER Richard North <rich.north@gmail.com>
 
 LABEL Description="This image can be used to create a sidekick container for recording videos of VNC sessions hosted in other containers"
+LABEL maintainer="Richard North <rich.north@gmail.com>"
 
 RUN apt-get update && apt-get install -y \
   python-pip python-dev \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && pip install vnc2flv \
+  && rm -r /tmp/*
 
-RUN pip install vnc2flv
+# INSTALL ffmpeg
+ADD https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz .
```

Comment:
I was thinking it might be possible to install this via apt, but probably ffmpeg contains non-codecs and is therefore not in Debian package repositories?

Reply:
Indeed It was the easiest solution, but I had some difficulties at the beginning to install **ffmpeg** via **apt**, as stated in [Debian wiki page](https://wiki.debian.org/ffmpeg), it is not available for Debian 8.
...So I thought of two solutions: 
- Compile and install it from the [source code](https://github.com/FFmpeg/FFmpeg)
- Or Use the available [static version](https://johnvansickle.com/ffmpeg/), which is mentioned in [ffmpeg official download page](https://ffmpeg.org/download.html) under **Linux Static Builds**.